src/main/java/org/springframework/samples/petclinic/PetClinicApplication.java

```diff
@@ -26,6 +27,7 @@
  *
  */
 @SpringBootApplication
+@EnableScheduling
```

Comment:
Is this annotation necessary? Same applies to the @component one below

Reply:
According to the Spring documents, it's necessary for scheduled tasks to run properly